.circleci/config.yml

```diff
@@ -237,13 +266,7 @@ jobs:
             . env/bin/activate
             cd tests/
             pip install -r requirements.txt
-            coverage run --source=dimod --append -m unittest
-      - run: &unix-codecov
-          name: codecov
-          command: |
-            . env/bin/activate
-            cd tests/
-            codecov
+            python -m unittest
```

Comment:
```suggestion
            python -m unittest discover
```

Reply:
Why?

Reply:
Is `discover` implied? I see you use it below.

Reply:
Oh, looks like it is implied. Ignore the suggestion then please.

Reply:
It is indeed. Agree I should be consistent.